src/main/java/com/softserve/repository/impl/ScheduleRepositoryImpl.java

```diff
@@ -433,4 +450,13 @@ public Long countByLessonIdPeriodIdEvenOddDayOfWeek(Long lessonId, Long periodId
                 .setParameter("evenOdd", evenOdd)
                 .getSingleResult();
     }
+
+    @Override
+    public List<Schedule> getAllOrderedByRoomsDaysPeriods(Long semesterId) {
```

Comment:
just name it getAllOrdered
after implementation of sortOrder, you will not have to change the method name :)